ECOSYSTEM.md

```diff
@@ -30,7 +37,10 @@ Our current partners and other areas of ownership:
 * **[Software Mansion](https://swmansion.com/):** Maintain core infrastructure including JSC, Animated, and other popular third-party plugins and organizes [App.js Conf](https://appjs.co/)
 * **[Wix.com](https://wix.engineering/open-source):** Maintains a variety of React Native open source projects ([see all](https://github.com/orgs/wix/repositories?q=react-native)), including:  [Detox](https://wix.github.io/Detox/) end-to-end testing library for React Native apps, [RN UILib](https://wix.github.io/react-native-ui-lib/), [RN Navigation](https://wix.github.io/react-native-navigation/), [RN Calendars](https://wix.github.io/react-native-calendars/) and [RN Notifications](https://github.com/wix/react-native-notifications).
 
-This list may fluctuate in response to newcomers who meet our partner definition. In terms of open source work, pull requests from partners are commonly prioritized. When you are contributing to React Native, you'll most likely meet somebody who works at one of the partner companies and who is a core contributor:
+### Incubating partners:
+* There are no current incubating partners. 
+
+When you are contributing to React Native, you'll most likely meet somebody who works at one of the partner companies and who is a core contributor.
```

Comment:
nit: I'm not sure I understand why this phrase is here/what purpose it serves; is it to help perspective partners understand how they can find someone to champion them? 🤔